include/h2o/rand.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2014 DeNA Co., Ltd.
```

Comment:
Do you mind if the copyright gets written as `Copyright (c) 2016 David CARLIER`?

We generally do not request transfer of copyright when accepting a PR.